packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -322,6 +322,11 @@ class AppBar extends StatefulWidget implements PreferredSizeWidget {
   /// changes the [AppBar]'s size. A [SliverAppBar] in a [CustomScrollView]
   /// changes the [AppBar]'s height when scrolled.
   ///
+  /// While including [SliverAppBar.flexibleSpace] it is important to mention
+  /// [expandedHeight] property because if a [flexibleSpace] widget is specified
+  /// [expandedHeight] should be big enough to accommodate whatever that widget
+  /// contains.
+  ///
```

Comment:
We can simplify this a bit, and let's actually move it to the 'flexibleSpace' of the `SliverAppBar` class where it is relevant.
The `AppBar` does not have an `expandedHeight`.
```suggestion
  /// When using [SliverAppBar.flexibleSpace], the [SliverAppBar.expandedHeight]
  /// must be large enough to accommodate the [SliverAppBar.flexibleSpace] widget.
  ///
```

Reply:
Sure @Piinks I will make the required changes.

Reply:
Are you talking about [this](https://github.com/flutter/flutter/blob/1ad9baa8b99a2897c20f9e6e54d3b9b359ade314/packages/flutter/lib/src/material/app_bar.dart#L957), but it is present in [appBar](https://github.com/flutter/flutter/blob/1ad9baa8b99a2897c20f9e6e54d3b9b359ade314/packages/flutter/lib/src/material/app_bar.dart#L1056).
And I have changed the line for above, flexibleSpaceBar.